modules/data.land/R/extract_soil_nc.R

```diff
@@ -15,35 +15,39 @@
 #'    PEcAn.data.land::extract_soil_gssurgo(outdir,lat,lon)
 #' }
 #' @author Hamze Dokoohaki
-
-extract_soil_gssurgo<-function(outdir,lat,lon){
+#' @import XML dplyr purrr
```

Comment:
I'm pretty sure this line can be deleted as  `@import` is redundant with the use of namespaces, and that we prefer that people use namespaces instead.   @infotroph @ashiklom did I get that right?

Reply:
Yes, `@import` is redundant with namespace. But do scour the `devtools::check` output after removing them to make sure there aren't any forgotten un-namespaced calls -- look for "...no visible global function definition...".

Reply:
it said:
```   Consider adding
     importFrom("grDevices", "dev.off", "pdf")

     importFrom("graphics", "abline", "boxplot", "hist", "layout", "legend",
                "lines", "pairs", "par", "plot", "points", "text", "title")
     importFrom("stats", "coef", "complete.cases", "cor", "cov", "density",
                "dgamma", "dnorm", "lm", "median", "na.omit", "pnorm",
                "predict.lm", "qnorm", "quantile", "rgamma", "rnorm",
                "runif", "sd", "setNames", "var")

     importFrom("utils", "data", "read.csv", "read.fwf",
                "setTxtProgressBar", "txtProgressBar", "write.table")

   to your NAMESPACE file.

Reply:
Those are all base R packages, so I'm less obsessive about adding the namespaces for all of those. It doesn't look like you're not missing any other non-base packages, which is good

Reply:
@para2x @mdietze Yeah, these should be cleaned up eventually but aren't hurting much in the meantime. But do be aware that these are R CHECK's *guesses* -- usually but not always right -- about where the unimported functions / variables come from. For example, We wouldn't want to import `stats::median` if the message came from `somePrecomputedTable %>% select(ID, median, sd))`.